signer/migrations/0006__remove_default_user.sql

```diff
@@ -0,0 +1,13 @@
+-- Delete default user if it exists
+DO
+$do$
+BEGIN
```

Comment:
I'm not sure we should do this, because it might blindly delete `sbtc_user` from a live signer if they ended up using that name.

Reply:
Good point, removed the script directly -- then we can communicate this to the current signers and let them decide if they want to remove the default user or not.